src/plugins/performance.ts

```diff
@@ -72,9 +67,9 @@ class PerformanceProfiler implements IComponent {
       formattedTiming = mapProperties(formattedTiming, (name: string, value) => {
         return (formattedTiming[name] === 0) ? 0 : Math.round(formattedTiming[name] - formattedTiming.navigationStart);
       }, false);
-      addEvent(TimingEventType, { timing: formattedTiming });
+      addEvent("NavigationTiming", { timing: formattedTiming });
```

Comment:
A breaking change to the schema. Please add a major version update to the version information.